chaincode/tuple.go

```diff
@@ -256,3 +294,11 @@ func determineTupleStatus(db *LedgerDB, tupleStatus, computePlanID string) (stri
 	}
 	return tupleStatus, nil
 }
+
+func createModelIndex(db *LedgerDB, tupleKey, modelHash string) error {
+	err := db.CreateIndex("tuple~modelHash~key", []string{"tuple", modelHash, tupleKey})
+	if err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
Yes indeed. To you think I should removed it or used it.
On one side it will not change the number of line but on the bright side this index name will not be copied for each creation (so 5 time in actual the base code)

Reply:
Done